rest_models/backend/compiler.py

```diff
@@ -974,19 +977,40 @@ def execute_sql(self, return_id=False, chunk_size=None):
         opts = query.get_meta()
         can_bulk = not return_id and self.connection.features.has_bulk_insert
 
+        def build_data(obj):
```

Comment:
1. refactored the duplicated code into a local function
2. changed this code to prevent FileField to be sent as normal data
3. added a function to handle FileField in the post(files=) part.